test/functionalTest/cases/4149_pagination_for_request_forwarding/pagination_without_request_forwarding.test

```diff
@@ -0,0 +1,365 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+pagination
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create entity A1/T in CB
+# 02. Create entity A2/T in CB
+# 03. Create entity A3/T in CB
+# 04. Create entity A4/T in CB
+# 05. Create entity A5/T in CB
+# 06. GET /v2/entities with limit=5
+# 07. GET /v2/entities with offset=2
+# 08. GET /v2/entities with options=count
+# 09. GET /v2/entities with limit=5 and offset=2
+# 10. GET /v2/entities with limit=2 and options=count
+# 11. GET /v2/entities with offset=2 and options=count
```

Comment:
According to this script, this .test doesn't do anything related with CPr. Thus, it doesn't seem to be related with issue #4149. Or maybe I'm missing something?

Reply:
Yes, these FTs are not related with CPr. These FTs are related with pagination in normal case.
We have included these FTs, to confirm that existing pagination functionality (Without Request Forwarding) is also working fine.

Reply:
Yes, these FTs are not related with CPr. These FTs are related with pagination in normal case.
We have included these FTs, to test that existing pagination functionality (Without Request Forwarding) is also working fine.

Reply:
Ok. It doesn't hurt :)

Let's keep the test. Although maybe it's better to use a better name. Suggestion: pagination_check_regular_functionality_does_not_break.test

Reply:
Fixed in [1e47b07](https://github.com/telefonicaid/fiware-orion/pull/4275/commits/1e47b077ce86f15d09438331b764710751bf1910)